.gitmodules

```diff
@@ -1,19 +1,19 @@
 [submodule "src/coreComponents/LvArray"]
 	path = src/coreComponents/LvArray
-	url = ../LvArray.git
+	url = https://github.com/GEOSX/LvArray.git
 [submodule "src/cmake/blt"]
 	path = src/cmake/blt
-	url = ../../LLNL/blt.git
+	url = https://github.com/LLNL/blt.git
 [submodule "src/coreComponents/constitutive/PVTPackage"]
 	path = src/coreComponents/constitutive/PVTPackage
-	url = ../PVTPackage.git
+	url = https://github.com/GEOSX/PVTPackage.git
 [submodule "integratedTests"]
 	path = integratedTests
-	url = ../integratedTests.git
+	url = git@github.com:GEOSX/integratedTests.git 
 [submodule "src/coreComponents/mesh/PAMELA"]
 	path = src/coreComponents/mesh/PAMELA
-	url = ../PAMELA.git
+	url = https://github.com/GEOSX/PAMELA.git
 	shallow = true
 [submodule "src/coreComponents/fileIO/coupling/hdf5_interface"]
 	path = src/coreComponents/fileIO/coupling/hdf5_interface
-	url = ../hdf5_interface.git
+	url = https://github.com/GEOSX/hdf5_interface.git
```

Comment:
When pulling submodules in Gitlab CI, the relative paths are relative to GEOSX's Gitlab path (https://lc.llnl.gov/gitlab/settgast/GEOSX) instead of Github path (https://github.com/GEOSX/GEOSX).

I prefer we pulled Github over Gitlab repositories because the Gitlab repositories are supposed to mirror the Github repositories every 30 minutes or so, but this mirroring has failed on LC in the past and requires manually going into the project settings to restart it.

In blt's case, the Gitlab location is https://lc.llnl.gov/gitlab/blt/blt, but the Github location is https://github.com/LLNL/blt, so a relative path would not work for both cases.

Reply:
Couldn't you apply some `git` scripting to properly redefine those urls?
Something like https://git-scm.com/docs/git-submodule#Documentation/git-submodule.txt-set-url--ltpathgtltnewurlgt  ?